src/main/webapp/app/layouts/main/main.component.ts

```diff
@@ -22,15 +23,17 @@ export class JhiMainComponent implements OnInit {
     }
 
     private async setupAnalytics() {
-        const profileInfo = await this.profileService.getProfileInfo();
-
-        if (profileInfo.inProduction && window.location.host === 'artemis.ase.in.tum.de') {
-            // only Track in Production Environment
-            this.angularticsPiwik.startTracking();
-        } else {
-            // Enable Developer Mode in all other environments
-            this.angulartics.settings.developerMode = true;
-        }
+        this.profileService.getProfileInfo().subscribe((profileInfo: ProfileInfo) => {
+            if (profileInfo) {
+                if (profileInfo.inProduction && window.location.host === 'artemis.ase.in.tum.de') {
```

Comment:
I think we can simplify this and remove one level of `if` conditions. Something like this maybe?
```suggestion
                if (profileInfo && profileInfo.inProduction && window.location.host === 'artemis.ase.in.tum.de') {
```

Reply:
good point. i'll change it